Sources/Conformance/conformance.pb.swift

```diff
@@ -245,18 +245,24 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
 
   init() {}
 
-  public mutating func _protoc_generated_decodeField<T: SwiftProtobuf.FieldDecoder>(setter: inout T, protoFieldNumber: Int) throws {
-    switch protoFieldNumber {
-    case 1, 2: try payload.decodeField(setter: &setter, protoFieldNumber: protoFieldNumber)
-    case 3: try setter.decodeSingularField(fieldType: Conformance_WireFormat.self, value: &requestedOutputFormat)
+  public mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
```

Comment:
Something I've been thinking about lately—instead of having so much duplicated generation code for messages with storage and messages without (and having these methods on both the message and the storage), we should just generate everything in the message and have it reach into the storage as needed to access the field properties. This would mean methods like this, which are currently generated differently in each case, could instead be implemented in a protocol extension.